src/entities/models/dao.py

```diff
@@ -16,14 +15,25 @@ class AuditMixin(object):
 
 class SblTypeMappingDao(Base):
     __tablename__ = "fi_to_type_mapping"
+    version: Mapped[int] = mapped_column(nullable=False, default=0)
+    __mapper_args__ = {"version_id_col": version, "version_id_generator": False}
```

Comment:
I'd love to understand what this is doing.  I know _mapper_args__ is used for polymorphic tables which makes sense so we can use the same DAO for both the normal and history table data.  But what magic is happening here??

Reply:
I was initially using the builtin version id generator, but have since switched over to generating via code, so added the `"version_id_generator": False` bit, this can be removed, but wanted to leave in as a reference for possible versioning of simpler tables.